### DISCUSSION TITLE
Cursor Pagination

### DISCUSSION THREAD

**Discussion by nessimabadi** [rappasoft/laravel-livewire-tables#838] [Q&A]
Is there a way paginate the results via cursor paginator instead of the default offset pagination?
I don't find anything on the docs, but I see that the WithPagination trait uses the WithPagination trait from Livewire, which handles cursor pagination.

Any help would be appreciated, thanks

**Comment by TheOWler** [comment]
hi, did you manage to solve your problem ?

**Reply by nessimabadi** [reply]
No, couldn't find a way, still using the basic pagination
